client/src/lib/user/state/state.ts

```diff
@@ -24,8 +24,14 @@ type PinnedSession = {
   expires: Date;
 };
 
+export type SavedCollection = {
+  id: string;
+  statedAt: string;
+};
+
 export type UserState = {
   pinnedSessions?: Array<PinnedSession>;
+  savedCollections?: Array<SavedCollection>;
```

Comment:
Is it really saved? Isn't it pinned or started?

Reply:
enrolledCollections?

Reply:
pinned perhaps, consistent with where pinned sessions end up kind of